egs/voxceleb/v2/run.sh

```diff
@@ -18,24 +18,27 @@ vaddir=`pwd`/mfcc
 
 
 voxceleb1_trials=data/voxceleb1_test/trials
```

Comment:
is data/voxceleb1_test/trials an input to the script?  do we need to make it downlaodable?
you might want to add a comment to e.g. 
```
# change this to where you have downloaded (or want the script to
# download) the VoxCeleb corpus).
```
(obviously edited for accuracy, I don't know if it auto-downloads).

Reply:
`local/make_voxceleb1.pl` creates `data/voxceleb1_test/trials` by converting the trials file provided by the authors of VoxCeleb to a format usable in Kaldi. It tries to download it from the official URL if it does not already exist in the VoxCeleb1 data distribution. Perhaps it would be possible to mirror that file on openslr.org?

Reply:
If @david-ryan-snyder has time to deal with it, mirroring that sounds like a good idea.

Reply:
Sure, we can mirror it, and download the file from the mirror instead.